One e2e job failed with this error; I restarted it:

```
Minifying and creating sourcemap for third_party/generated/js/third_party.js

/bin/sh: 1: /home/travis/build/oppia/oppia/../oppia_tools/node-10.18.0/bin/node: not found

Traceback (most recent call last):

  File "/opt/python/2.7.15/lib/python2.7/runpy.py", line 174, in _run_module_as_main

    "__main__", fname, loader, pkg_name)

  File "/opt/python/2.7.15/lib/python2.7/runpy.py", line 72, in _run_code

    exec code in run_globals

  File "/home/travis/build/oppia/oppia/scripts/build.py", line 1271, in <module>

    main()

  File "/home/travis/build/oppia/oppia/scripts/build.py", line 1258, in main

    minify_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)

  File "/home/travis/build/oppia/oppia/scripts/build.py", line 528, in minify_third_party_libs

    THIRD_PARTY_JS_FILEPATH, MINIFIED_THIRD_PARTY_JS_FILEPATH)

  File "/home/travis/build/oppia/oppia/scripts/build.py", line 232, in _minify_and_create_sourcemap

    subprocess.check_call(cmd, shell=True)

  File "/opt/python/2.7.15/lib/python2.7/subprocess.py", line 190, in check_call

    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command '/home/travis/build/oppia/oppia/../oppia_tools/node-10.18.0/bin/node node_modules/uglify-js/bin/uglifyjs third_party/generated/js/third_party.js -c -m --source-map includeSources,url='third_party.min.js.map' -o third_party/generated/js/third_party.min.js ' returned non-zero exit status 127

kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]

scripts/run_e2e_tests.sh: line 53: kill: (5258) - No such process

Done!

The command "bash scripts/run_e2e_tests.sh --suite=explorationImprovementsTab --prod_env" failed. Retrying, 2 of 3.
```

--- a/core/templates/dev/head/pages/admin-page/config-tab/admin-config-tab.directive.ts
+++ b/core/templates/dev/head/pages/admin-page/config-tab/admin-config-tab.directive.ts
@@ -57,10 +57,9 @@ angular.module('oppia').directive('adminConfigTab', [
             return;
           }
 
-          AdminConfigTabBackendApiService.revertConfigProperty({
-            action: 'revert_config_property',
-            config_property_id: configPropertyId
-          }).then(function() {
+          AdminConfigTabBackendApiService.revertConfigProperty(
+            configPropertyId
+          ).then(function() {
             ctrl.setStatusMessage('Config property reverted successfully.');
             ctrl.reloadConfigProperties();
           }, function(errorResponse) {
@@ -86,10 +85,9 @@ angular.module('oppia').directive('adminConfigTab', [
               ctrl.configProperties[property].value);
           }
 
-          AdminConfigTabBackendApiService.saveConfigProperties({
-            action: 'save_config_properties',
-            new_config_property_values: newConfigPropertyValues
-          }).then(function() {
+          AdminConfigTabBackendApiService.saveConfigProperties(
+            newConfigPropertyValues
+          ).then(function() {
             ctrl.setStatusMessage('Data saved successfully.');
             AdminTaskManagerService.finishTask();
           }, function(errorResponse) {

--- a/core/templates/dev/head/pages/admin-page/services/admin-config-tab-backend-api.service.ts
+++ b/core/templates/dev/head/pages/admin-page/services/admin-config-tab-backend-api.service.ts
@@ -21,12 +21,18 @@ require('pages/admin-page/admin-page.constants.ajs.ts');
 angular.module('oppia').factory('AdminConfigTabBackendApiService', [
   '$http', 'ADMIN_HANDLER_URL',
   function($http, ADMIN_HANDLER_URL) {
-    var _revertConfigProperty = function(data) {
-      return $http.post(ADMIN_HANDLER_URL, data);
+    var _revertConfigProperty = function(configPropertyId) {
+      return $http.post(ADMIN_HANDLER_URL, {
+        action: 'revert_config_property',
+        config_property_id: configPropertyId
+      });
     };
 
-    var _saveConfigProperties = function(data) {
-      return $http.post(ADMIN_HANDLER_URL, data);
+    var _saveConfigProperties = function(newConfigPropertyValues) {
+      return $http.post(ADMIN_HANDLER_URL, {
+        action: 'save_config_properties',
+        new_config_property_values: newConfigPropertyValues
+      });
     };
 
     return {

